Resources/Controllers/BottomSheet/README.md

```diff
@@ -0,0 +1,159 @@
+#  BottomSheet
```

Comment:
Why the name change? [Wizard](https://docs.microsoft.com/en-us/windows/win32/uxguide/win-wizards) is a common name used for a multi-step user interface.